promises.cf.in

```diff
@@ -213,7 +213,7 @@ bundle common cfe_internal_inputs
         comment => "This policy configures external watchdogs to ensure that
                     cf-execd is always running.";
 
-    enterprise_edition.(policy_server|am_policy_hub)::
+    any::
```

Comment:
This change seemed to have a positive effect on the test outcome

Reply:
I wonder if we can undo this and add class guards to the methods promises instead.